src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -187,6 +187,12 @@ function assertValidProps(component, props) {
       'those nodes are unexpectedly modified or duplicated. This is ' +
       'probably not intentional.'
     );
+    warning(
+      props.onFocusIn == null &&
+      props.onFocusOut == null,
+      'Props onFocusIn and onFocusOut are not supported. Use onFocus and ' +
+      'onBlur events instead because they are normalized across browsers.'
```

Comment:
Users can use `onFocus` in leu of `onFocusIn` because the React variant bubbles.  We actually deviate from the standard here, so it's more than just "normalization across browsers".

Perhaps:
 > React uses `onFocus` and `onBlur` instead of `onFocusIn` and `onFocusOut`.  All React events are normalized to bubble, so `onFocusIn` and `onFocusOut` are not needed/supported by React.